op-service/sources/testdata/gen.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
```

Comment:
We should mark this file as executable.  And also I'd recommend adding:
```suggestion
#!/bin/bash
set -euo pipefail

SOURCE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
cd "$SOURCE_DIR"
```

so the script doesn't just ignore errors and continue, and so it can be run from any directory